llvm/tools/llvm-profgen/ProfiledBinary.cpp

```diff
@@ -707,6 +703,19 @@ void ProfiledBinary::disassemble(const ELFObjectFileBase *Obj) {
     }
   }
 
+  if (!AddrsWithInvalidInstruction.empty()) {
```

Comment:
It seems like we print aggregate after detailed warning for AddrsWithMultipleSymbols. Can we be consistent here?